js/emailvalidation.js

```diff
@@ -0,0 +1,47 @@
+emailjs.init("Your_User_ID"); // Replace with your actual EmailJS User ID
```

Comment:
Export the function and use it where it used before this fn

Reply:
I didn't get u.

Reply:
@varshith257 could u explain clearly.